src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py

```diff
@@ -260,7 +265,8 @@ def getSiteStatus(self):
             ssbStatus = ssbState[site]['prod_status']
             wmcoreStatus = self.getState(str(ssbStatus))
             if not wmcoreStatus:
-                logging.error("Unknown status '%s' for site %s, please check SSB", ssbStatus, site)
+                logging.warning("Site %s has an unknown SSB status '%s'. Skipping it!", site, ssbStatus)
+                ssbStatus.pop(site, None)
```

Comment:
@amaltaro I think this should be `ssbState.pop(site, None)`?

Reply:
And I think you're correct!